recipe/meta.yaml

```diff
@@ -64,6 +65,9 @@ test:
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}
     - sysroot_linux-64 2.17  # [linux64]
+    # At this point we are still inside the 11.0 docker, so needs to avoid pulling
+    # down a newer runtime, otherwise the linker would complain
+    - cudatoolkit 11.0  # [linux and cuda_compiler_version == "11.0"]
```

Comment:
I need to investigate why we have to do this on Linux but not on Windows.

Reply:
I noticed the effect of adding `nvcc` to `ignore_run_exports_from` is that at build time, `cudatoolkit` is no longer installed. Although it's not really used at build time, I don't think this behavior is correct.